Tasks/VsTest/VSTest.ps1

```diff
@@ -68,7 +69,7 @@ if ($testAssembly.Contains("*") -Or $testAssembly.Contains("?"))
 else
 {
     Write-Verbose "No Pattern found in solution parameter."
-    $testAssemblyFiles = ,$testAssembly
+    $testAssemblyFiles += ,$testAssembly.Split(";")
```

Comment:
Find files supports it by escaping a semicolon with a semicolon. E.g. if you actually have a semicolon in your file path, then you would need to escape it before passing off to find-files: "c:\myfolder\my;;file"

I've updated the docs here: https://github.com/Microsoft/vsts-task-lib/blob/master/powershell/Docs/FullHelp/Find-VstsFiles.md

It should probably be documented somewhere else as well, since that is really the docs for the SDK, not the end-user/consumer of the task.

That said, I haven't heard the issue come up before that folks are having trouble because their file path contains a semicolon.